packages/sdk/package.json

```diff
@@ -50,13 +54,6 @@
   "devDependencies": {
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/instrumentation": "~0.43.0",
-    "@opentelemetry/instrumentation-bunyan": "^0.32.1",
-    "@opentelemetry/instrumentation-fs": "^0.8.1",
-    "@opentelemetry/instrumentation-http": "~0.43.0",
-    "@opentelemetry/instrumentation-mysql2": "^0.34.1",
-    "@opentelemetry/instrumentation-pg": "^0.36.1",
-    "@opentelemetry/instrumentation-pino": "^0.34.1",
-    "@opentelemetry/instrumentation-winston": "^0.32.1",
```

Comment:
This was to be able to import types but now we already have access to those types from the `instrumentations` package's dependencies.